pkg/cli/image/append/append.go

```diff
@@ -84,9 +86,14 @@ var (
 		oc image append --from-dir v2 --to myregistry.com/myimage:latest layer.tar.gz
 
 		# Add a new layer to a multi-architecture image for an os/arch that is different from the system's os/arch
-		# Note: Wildcard filter is not supported with append. Pass a single os/arch to append
 		oc image append --from docker.io/library/busybox:latest --filter-by-os=linux/s390x --to myregistry.com/myimage:latest layer.tar.gz
 
+		# Add a new layer to a multi-architecture image for all the os/arch manifests when keep-manifest-list is specified
+		oc image append --from docker.io/library/busybox:latest --keep-manifest-list --to myregistry.com/myimage:latest layer.tar.gz
+
+		# Add a new layer to a multi-architecture image for all the os/arch manifests that is specified by the filter, while preserving the manifestlist
+		oc image append --from docker.io/library/busybox:latest --filter-by-os=linux/s390x --keep-manifest-list --to myregistry.com/myimage:latest layer.tar.gz
```

Comment:
does `--filter-by-os` accepts multiple os/arch to be specified? The previous comment was very clear about it only accepting one, as well as not allowing wild cards. I think the new comment should be too.

Reply:
ah..yes..added that statement back.